src/nvim/api/vim.c

```diff
@@ -1481,13 +1481,13 @@ Window nvim_open_win(Buffer buffer, Boolean enter, Dictionary config,
   if (enter) {
     win_enter(wp, false);
   }
+  if (win_valid(wp) && buffer > 0) {
```

Comment:
so the win_enter() just above, could trigger autocmds which delete the window or buffer. Right? would be worth a 1-line comment.

Reply:
Yup, autocommands from `win_enter` can also close the float #11383. I'll add a comment. :+1: